Jenkinsfile_CNP

```diff
@@ -10,10 +10,97 @@ def component = "case-payment-orders-api"
 
 GradleBuilder builder = new GradleBuilder(this, product)
 
+def secrets = [
+  'rpx-${env}': [
+    secret('mc-idam-client-secret', 'BEFTA_OAUTH2_CLIENT_SECRET_OF_XUIWEBAPP')
+  ],
+  'ccd-${env}': [
+    // BEFTA user: Common_User_For_Request
+    secret('ccd-caseworker-autotest-email', 'CCD_CASEWORKER_AUTOTEST_EMAIL'),
+    secret('ccd-caseworker-autotest-password', 'CCD_CASEWORKER_AUTOTEST_PASSWORD')
+
+  ],
+  's2s-${env}': [
+    secret('microservicekey-xui-webapp', 'BEFTA_S2S_CLIENT_SECRET_OF_XUI_WEBAPP')
+  ]
+]
+
+static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
+  [$class     : 'AzureKeyVaultSecret',
+    secretType : 'Secret',
+    name       : secretName,
+    version    : '',
+    envVariable: envVar
+  ]
+}
+
+def vaultOverrides = [
```

Comment:
i think this is not required any more. we have to cleanup in other apps soon.

Reply:
Thanks, removed in 724fb6289123e7a62d84aa0bb7ac3181934cf454.